auditlog_security/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Therp B.V. <https://www.therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': "Audit Log Permissions",
+    'version': "11.0.1.0.1",
```

Comment:
We start with `11.0.1.0.0`, and I think a better name would be `auditlog_fields`.

But leave this for the moment, let's think about the functionality first. If there are no downsides to using it (eg. it won't become slower or more buggy), then I suggest we just move it directly into the main `auditlog` module.